### ISSUE TITLE
A question regarding the importance metric

### ISSUE THREAD

**Issue by nzomi** [visresearch/SDMPrune#2]
Hi authors, since SDMPrune calculates importance by minimizing the difference between the pruned and original model logits, does a smaller importance value correspond to a smaller increase in loss? If that is the case, why do you select the top-k channels with the highest importance values instead of the ones with the lowest?

the goal is $min_{W'}|L(D|W') - L(D|W)|$, 
and you defined $I_i = |L(D|W_d=0, W_g=0, W_u=0) - L(D|W_d, W_g, W_u)|$,
Therefore, a lower importance corresponds to a smaller increase in error.

**Comment by nzomi** [comment]
mb, that means if prune the low-importance channels, the performance won’t be significantly affected, so keep the topk channels
